opencog/atomspace/AtomTable.h

```diff
@@ -93,7 +93,7 @@ class AtomTable
     // handle uuid and the actual atom pointer (since they are stored
     // together).  To some degree, this info is duplicated in the Node
     // and LinkIndex below; we have this here for convenience.
-    std::set<Handle, handle_less> _atom_set;
+    std::unordered_set<Handle, handle_hash> _atom_set;
 
```

Comment:
Makes sense. I tried to lay out the issues on the list before submitting this. I wrote the benchmark changes while researching the effects of the change as I wanted to be able to say how much this change affected the Python binding performance. Still, I could have submitted the two parts separately certainly and will do so in the future.